py/selenium/webdriver/remote/remote_connection.py

```diff
@@ -135,6 +135,9 @@ class RemoteConnection(object):
 
     Communicates with the server using the WebDriver wire protocol:
     https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol"""
+
+    timeout = socket._GLOBAL_DEFAULT_TIMEOUT
```

Comment:
I look forward to the updated version!

David Burns
Email: david.burns@theautomatedtester.co.uk
URL: http://www.theautomatedtester.co.uk/

On Wed, May 27, 2015 at 11:03 AM, Sergey Chipiga <notifications@github.com>
wrote:

> In py/selenium/webdriver/remote/remote_connection.py
> <https://github.com/SeleniumHQ/selenium/pull/324#discussion_r31118899>:
>
> > @@ -135,6 +135,9 @@ class RemoteConnection(object):
> >
> >      Communicates with the server using the WebDriver wire protocol:
> >      https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol"""
> > +
> > +    timeout = socket._GLOBAL_DEFAULT_TIMEOUT
>
> @AutomatedTester <https://github.com/AutomatedTester>, thank you for you
> comment. And actually I need to update this pull-request, because in our
> custom selenium package, we found, that it's not enough to support socket
> timeout, because it need to catch socket.error too. I will update
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SeleniumHQ/selenium/pull/324/files#r31118899>.
>